### ISSUE TITLE
[BUG] Wrong python3 path for .deb package

### ISSUE THREAD

**Issue by cinghioGithub** [HorlogeSkynet/archey4#153]
When installing the latest release using the ```.deb``` package, the python binary path indicated in the installed ```/usr/bin/archey``` script is wrong and it brakes the execution. Instead of a system path a hard coded path is reported. I report the issue I had:

```
$ sudo apt install ./archey4_4.14.3.0-1_all.deb
$ archey
zsh: /usr/bin/archey: bad interpreter: /home/samuel/Documents/archey4/venv/bin/python3: no such file or directory
```

**Environment**

 - Version used : ```4.14.3.0```
 - Method of installation : ```.deb``` 
 - Operating system and version : Ubuntu 22.04.4

**Comment by HorlogeSkynet** [comment]
Thanks for opening this, it looks like venv messed up with packaging script. I'll look into this soon.

**Comment by HorlogeSkynet** [comment]
A month and more than 80 broken package downloads  and you're the only one who opened an issue about that... Thanks again, it has been fixed in `v4.14.3.0-2` packages (see https://github.com/HorlogeSkynet/archey4/releases/tag/v4.14.3.0 assets).

Bye :wave:
